Zappy_IA_Report/NN/Code/Tutorial4.py

```
@@ -0,0 +1,31 @@
+
+(train_data, train_labels), (test_data, test_labels) = data.load_data(num_words=88000)
+
+word_index = data.get_word_index()
+
+word_index = {k:(v+3) for k, v in word_index.items()}
+word_index["<PAD>"] = 0
+word_index["<START>"] = 1
+word_index["<UNK>"] = 2
+word_index["<UNUSED>"] = 3
+
+reverse_word_index = dict([(value, key) for (key, value) in word_index.items()])
```

Ruff (0.11.9)

Unnecessary list comprehension (rewrite as a dict comprehension)

Rewrite as a dict comprehension

(C404)